### DISCUSSION TITLE
[Feature request / question] Support for web speech api

### DISCUSSION THREAD

**Discussion by arthur-er** [remotion-dev/remotion#557] [Q&A]
Hello!

Its possible to use the Web Speech API to TTS dynamically generated audio for the video?

Thanks for your time. 💛

**Comment by JonnyBurger** [comment]
Hi @arthur-er!

The problem with it (also discussed here: https://github.com/remotion-dev/remotion/issues/249) is that you cannot grab the audio of the Web Speech API and record it, therefore Remotion cannot access it during the render process 😌

Therefore we made a template that uses the Azure TTS API instead, which you can access here: https://github.com/FelippeChemello/Remotion-TTS-Example

**Comment by tomByrer** [comment]
Could record what is output by a comptuer's speakers via an 'headphones to mic' cable.  Then use [recording API](https://gist.github.com/meziantou/edb7217fddfbb70e899e) to record the the mic input.
